application/common/models/Method.php

```diff
@@ -177,4 +174,33 @@ public static function deleteExpiredUnverifiedMethods()
             }
         }
     }
+
+    /**
+     * @param integer $userId
+     * @param string $value
+     * @return Method
+     * @throws ConflictHttpException
+     * @throws ServerErrorHttpException
+     */
+    public static function create($userId, $value)
```

Comment:
This may be more of a change that you want to make at the moment, but it seems like this function's name doesn't quite line up with what it does. Do you think it would be worth renaming this to `findOrCreate(...)`?